### ISSUE TITLE
Missing Kotlin examples

### ISSUE THREAD

**Issue by djensen47** [ebean-orm/website-source#39]
There are a many places where both Java and Kotlin examples are provided. This is great! However there are a few places where there are no Kotlin examples provided. (I'll be updating this list as I discover them).

- https://ebean.io/docs/query/dtoquery
- https://ebean.io/docs/query/select

**Comment by rbygrave** [comment]
Yes.  We have a better java/kotlin mechanism now and that whole docs / query section needs updating for that.
